Firmware/MotorControl/axis.hpp

```diff
@@ -45,11 +47,14 @@ class Axis {
         bool startup_sensorless_control = false; //<! enable sensorless control after calibration/startup
         bool enable_step_dir = false; //<! enable step/dir input after calibration
                                     //   For M0 this has no effect if enable_uart is true
+        bool use_enable_pin = false;
+        bool enable_pin_active_low = true;
```

Comment:
Wouldn't a default of `false` be more expected by users?

Reply:
Hm yes, lets change this.